Player/Input.php

```diff
@@ -140,7 +141,12 @@ private function splitInput($input)
                     $val .= substr($line, \strlen($indent))."\n";
                 }
 
-                $lines[$current] .= ' '.$this->escapeValue($val);
+                $escaped = ' ' . $this->escapeValue($val);
+                if (\in_array('i', $modifiers, true)){
+                    $lines[$current] .= preg_replace('/(?<!\\\\)\\$\{\s*(\w+)\s*\}/', "' ~ $1 ~ '", $escaped);
```

Comment:
A last thing and I think we are good:
I believe `\w` should be replaced by the constant `Parser::REGEX_NAME`

Reply:
Fixed & covered by test.